react-native-tileview/index.js

```diff
@@ -0,0 +1,23 @@
+import { PropTypes } from 'react';
+import { requireNativeComponent, View } from 'react-native';
+
+var iface = {
+   name: 'Tileview',
+   propTypes: {
+      displayTiles: PropTypes.string,
+      ...View.propTypes // include the default view properties
+    }
+};
+
+module.exports = requireNativeComponent('RCTTileview', iface);
+// 'use strict';
```

Comment:
This commented code should be removed. But `'use strict';` should be at the top of this file